src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,9 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ Info }) => (
+  <div className="PostList">
+    {Info.map(post => (
```

Comment:
```suggestion
    {posts.map(post => (
```

Use camelCase for props